### DISCUSSION TITLE
Update the hospital data when user data is updated in real time

### DISCUSSION THREAD

**Discussion by Luson045** [Luson045/medi-connect#169] [Announcements]
So, I made this feature of appointment booking, where users can book appointment, here are the screenshots
1) User Panel  2)Hospital Panel
<div display="flex">
<img width="350" alt="image" src="https://github.com/user-attachments/assets/561c65b9-86d8-4428-b893-406656226e91">
<img width="350" alt="image" src="https://github.com/user-attachments/assets/a48dee9c-84cc-42dc-8c54-4d314d6f8233">
</div>

This is the user profile page
<img width="400" alt="image" src="https://github.com/user-attachments/assets/42f4c3ef-e1a2-4bfc-bd7e-e36ab69083c6">

now whenever the hospital deletes/ mark as done then it should be updated in the user database too

but it's not working, whenever the appointment is deleted from the hospital database it remains in the user database.

Fix this issue as described above

reference files - Frontend ---> HospitalPanal.jsx, Profile.jsx,   Backend------>index.js in hospital and user files ,   DB Models---> server/models

**Comment by sidhanshamil** [comment]
@Luson045 I would like to work on this, can you assign this issue #9 to me

**Comment by Luson045** [comment]
This is a level 3 issue so make sure you're familiar with MERN stack setup and setting up a mongodb database?

**Comment by sidhanshamil** [comment]
Sure, @Luson045! I'm comfortable with the MERN stack setup and setting up a MongoDB database. Thanks for assigning it to me, @Luson045! I'll start working on it ASAP.

**Comment by Luson045** [comment]
@sidhanshamil any update??

**Comment by Luson045** [comment]
@sidhanshamil complete this by tonight else someone else will be assigned this issue

**Comment by Luson045** [comment]
@sidhanshamil  you are unassigned and flagged from this repo!
